notebooks_tsqr/NightLog.ipynb

```diff
@@ -0,0 +1,755 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #55.            self.ignore_fields = list()
</pre><p>I was curious about whether or not to use <code>list()</code> or <code>[]</code> after seeing these lines, and it looks like performance wise we should use the latter </p><p><a href="https://stackoverflow.com/questions/5790860/whats-the-difference-between-and-vs-list-and-dict" target="_blank">https://stackoverflow.com/questions/5790860/whats-the-difference-between-and-vs-list-and-dict</a>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>